Packs/FeedMISP/Integrations/FeedMISP/FeedMISP.py

```diff
@@ -232,8 +232,10 @@ def build_params_dict(tags: List[str], attribute_type: List[str], limit: int, pa
         'limit': limit,
         'page': page
     }
+    if params.get("timestamp"):
+        params['attribute_timestamp'] = params.pop("timestamp")
```

Comment:
params is defined above this condition and you can see it never has a `timestamp` item